app/pkcs.py

```diff
@@ -26,17 +26,21 @@ class pkcs:
     attests = {}
     keys = {1: 1, 2: 2, 3: 3, 4: 4}
 
-    def __init__(self):
+    _yubikey_pin: str
+
+    def __init__(self, yubikey_pin: str):
         self.pkcs11 = PyKCS11.PyKCS11Lib()
         self.pkcs11.load("/usr/lib64/libykcs11.so.2")
 
+        self._yubikey_pin = yubikey_pin
+
     def getusersession(self, slot):
         print("User Open", slot)
         if slot not in self.sessions:
             # self.sessions[slot] = self.pkcs11.openSession(slot)
             # self.sessions[slot].login("123456")
             self.sessions[slot] = self.pkcs11.openSession(slot)
-            self.sessions[slot].login("123456")
+            self.sessions[slot].login(self._yubikey_pin)
```

Comment:
Would be nice if this would be a user input instead of configured variable. But for now this is already better.

Reply:
I agree, lets create a ticket for this.

Reply:
https://github.com/minvws/nl-uzipoc-yubisign/issues/29